sipyco/pc_rpc.py

```diff
@@ -603,7 +630,7 @@ async def _handle_connection_cr(self, reader, writer):
                 if not line:
                     break
                 reply = await self._process_and_pyonize(target,
-                    pyon.decode(line.decode()))
+                                                        pyon.decode(line.decode()))
```

Comment:
Please move all the style changes into a separate PR.

Reply:
Done, see #12